src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WorkingDayCountService.kt

```diff
@@ -19,4 +19,24 @@ class WorkingDayCountService(
     }
     return from.getDaysUntilInclusive(to).filter { it.isWorkingDay(bankHolidays) }.size
   }
+
+  fun calculateEndDate(startDate: LocalDate, numberOfWorkingDays: Int): LocalDate {
```

Comment:
@danlivings-dxw I wonder if this logic something you're also working on for booking creation.